scripts/clean-kind-only-e2e-test.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
```

Comment:
Could you create a make target for this script?

Reply:
sure. I think it will require changes in the way we are calculating the default value of KUBECONFIG in the global Makefile, however that is overdue, anyway.
Btw, I believe we should do a wide-scope clean up of our make targets together as a team. But that is out-of-scope of this PR.